runtime/server/connectors.go

```diff
@@ -106,6 +107,28 @@ func (s *Server) ListNotifierConnectors(ctx context.Context, req *runtimev1.List
 		}
 	}
 
+	// Connectors may be implicitly defined just by adding variables in the format "connector.<name>.<property>".
```

Comment:
Slack connector is defined even if bot token isn't there but only for webhooks

Reply:
Yeah the abstractions definitely get blurry with that kind of "anonymous" functionality. In this case, it's useful for the `ListNotifierConnectors` only to return connectors that have been configured somehow (with vars or by adding to `rill.yaml`). Will add a note in the PR about it.